gravitee-am-reporter/gravitee-am-reporter-mongodb/src/main/java/io/gravitee/am/reporter/mongodb/audit/MongoAuditReporter.java

```diff
@@ -115,7 +116,7 @@ public class MongoAuditReporter extends AbstractService<Reporter> implements Aud
     @Value("${management.mongodb.ensureIndexOnStart:true}")
 
     private boolean ensureIndexOnStart;
-    @Value("${management.mongodb.cursorMaxTime:60000}")
+    @Value("${management.mongodb.cursorMaxTime:28000}")
```

Comment:
Why the value at `28000` ?

Reply:
The curson max time has to be lower than socket timeout